packages/desktop-client/src/components/Modals.tsx

```diff
@@ -41,11 +44,17 @@ import { ScheduleDetails } from './schedules/ScheduleDetails';
 import { ScheduleLink } from './schedules/ScheduleLink';
 
 export function Modals() {
-  const modalStack = useSelector(state => state.modals.modalStack);
-  const isHidden = useSelector(state => state.modals.isHidden);
-  const accounts = useSelector(state => state.queries.accounts);
+  const modalStack = useSelector<State, ModalsState['modalStack']>(
+    state => state.modals.modalStack,
+  );
+  const isHidden = useSelector<State, ModalsState['isHidden']>(
+    state => state.modals.isHidden,
+  );
+  const accounts = useSelector<State, QueriesState['accounts']>(
+    state => state.queries.accounts,
+  );
   const { grouped: categoryGroups, list: categories } = useCategories();
-  const budgetId = useSelector(
+  const budgetId = useSelector<State>(
```

Comment:
Should this one also have the localPrefs id generic?

Reply:
Good catch! Updated :)